### PR TITLE
solution

### PR DESCRIPTION
**Pull request by crash2bash**
- [DEMO LINK](https://crash2bash.github.io/layout_colored-blocks/)
- [TEST REPORT LINK](https://crash2bash.github.io/layout_colored-blocks/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  padding: 0;
```
Comment (earthhuman41) [inline src/style.css]: body hasn't paddings

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.first-block,
+.second-block,
+.third-block {
```
Comment (earthhuman41) [inline src/style.css]: will be nice if you create common class for all blocks like .block

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  width: 100%;
```
Comment (earthhuman41) [inline src/style.css]: Block elements has 100% of the width of the parent container. Set width of 100% is redundant

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  height: 100px;
+}
+
+.first-block {
```
Comment (earthhuman41) [inline src/style.css]: block--red - better name then first-block

**Review by 8hokage — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 4 above)
Comment (crash2bash) [reply src/style.css]: Agree :)
All fixed

**Review by 8hokage — APPROVED** [review]

**Pull request closed without merging**